api/db/changelog/db.changelog-127-add-research-outcomes-workspace.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="nsaxena" id="changelog-127-add-research-outcomes-workspace">
+    <createTable tableName="workspace_research_outcomes">
```

Comment:
Any reason to do this as multiple changelogs? Rather than in one file?

Reply:
So from what i have learned from my past experience while adding date columns,  it's better/cleaner to each change-set in different file, just so in case there is an issue with one of them it will be easier to change/rollback.